test/symmetric.jl

```diff
@@ -1035,6 +1035,14 @@ end
         @test Aherm isa Hermitian
         @test Aherm.uplo == LinearAlgebra.char_uplo(uplo)
     end
+    @testset "hermitianpart for numbers" begin
+    @test hermitianpart(3 + 4im) == 3
+    @test hermitianpart(5) == 5
+    @test hermitianpart(2.5) == 2.5
+    @test hermitianpart(-1 + 0im) == -1
+    @test typeof(hermitianpart(3 + 4im)) == Int
```

Comment:
After the `float` change, this needs to be changed to

```suggestion
    @test typeof(hermitianpart(3 + 4im)) == Float64
```